src/common/transformations/src/transformations/common_optimizations/mark_precision_sensitive_divides.cpp

```diff
@@ -47,11 +47,20 @@ bool ov::pass::MarkPrecisionSensitiveDivides::run_on_model(const std::shared_ptr
 
         for (auto& input_value : curr_node->input_values()) {
             // continue searching
-            const auto& input_node = input_value.get_node();
-            if (visited.count(input_node))
+            const auto& input_node_ptr = input_value.get_node_shared_ptr();
+            if (visited.count(input_node_ptr.get()))
```

Comment:
You don't have to change `const auto& input_node = input_value.get_node();` because you still can cast it with `as_type_ptr<ngraph::op::util::MultiSubGraphOp>(input_node)`

Reply:
fixed